drupal/web/modules/custom/wunder_next/src/Commands/WunderNextCommands.php

```diff
@@ -85,15 +86,16 @@ public function setupUserAndConsumer(array $options = ['secret' => '']) {
       }
 
       $consumer->save();
-    } catch (EntityStorageException $e) {
+    }
+    catch (EntityStorageException $e) {
       return new CommandError('Could not create a new consumer.');
     }
 
-    $uuid = $consumer->uuid();
+    $client_id = $consumer->getClientId();
 
     // Output instructions to the user:
     $this->logger()->success(dt('User and consumer created. Add these two lines to the .env file for the Next application:'));
-    $this->output()->writeln("DRUPAL_CLIENT_ID=$uuid");
+    $this->output()->writeln("DRUPAL_CLIENT_ID=$client_id");
```

Comment:
We now output the client_id. Using the uuid here resulted in authentication failure by the frontend.